src/auth/auth.constants.ts

```diff
@@ -0,0 +1,3 @@
+export const AUTH_CONSTANTS = {
+  JWT_SECRET: 'Hossam',
```

Comment:
this can't be commited in codebase, it has to be an env variable, saved in .env untracked file

Reply:
I'm adding this as a task in trello.